src/main/resources/reference.conf

```diff
@@ -4,6 +4,11 @@ dataproc {
   applicationName = "firecloud:leonardo"
   dataprocDockerImage = "gcr.io/broad-dsde-prod/leonardo-notebooks:prod"
   jupyterServerName = "jupyter-server"
+  firewallRuleName = "leonardo-notebooks-rule"
+  networkTag = "leonardo"
+  # Unset to use project-level defaults. Note a VPC/subnet with the literal name "default" may not exist.
+  #vpcNetwork = "default"
```

Comment:
Should fix the issue in broad-dsde-dev: that project doesn't have a VPC/subnet named "default", but it works it you don't set it and let Google figure it out.